src/launcher/main.cc

```diff
@@ -1,13 +1,11 @@
 #include <string>
 #include <FL/Fl.H>
-#include <slog/slog.h>
 
 #include <Launcher.h>
 #include "RustInterface.h"
 
 int main(int argc, char* argv[]) {
-	SLOG_Init(SLOG_STDERR, "stracciatella-launcher.log");
-	SLOG_SetLevel(SLOG_WARNING, SLOG_WARNING);
+	Logger_Init("stracciatella-launcher.log");
```

Comment:
I would use the exe name `ja2-launcher.log` to match `ja2.log`.
Or change `ja2.log` to `stracciatella.log` to match this.